### ISSUE TITLE
There is no valid electron symbol debug file in self hosted sentry

### ISSUE THREAD

**Issue by theremoon-j** [getsentry/self-hosted#2811]
### Self-Hosted Version

23.12.1

### CPU Architecture

amd64

### Docker Version

25.0.2

### Docker Compose Version

2.24.5

### Steps to Reproduce

create a electron project in a new self hosted sentry, report a crash but cant find debug file. However the same case can find correct file by the same way in sentry.io.

### Expected Result

Same result like sentry.io
![image](https://github.com/getsentry/self-hosted/assets/39041796/9c6462ec-cf29-410a-bc96-cfc7d2b287df)
![image](https://github.com/getsentry/self-hosted/assets/39041796/d391b82f-2ddd-4f0d-bd73-4942adc8c269)

### Actual Result

Actually cant find the debug file.
![image](https://github.com/getsentry/self-hosted/assets/39041796/dd90ff83-1d5c-4562-88d4-03f40d3ee121)
![image](https://github.com/getsentry/self-hosted/assets/39041796/5e1c3c63-d761-4b3b-bc16-40068f90b2ce)
![image](https://github.com/getsentry/self-hosted/assets/39041796/95f84e68-8581-47b8-a8ff-9215710ffb6c)

**Comment by theremoon-j** [comment]
'Electron' is selected yet in 'BUILT-IN REPOSITORIES'.
![image](https://github.com/getsentry/self-hosted/assets/39041796/23e2355f-ad89-4c4e-afa1-7e1760a2e589)

**Comment by azaslavsky** [comment]
In both cases, you are uploading the debug files via the `sentry-cli`, yes?

**Comment by theremoon-j** [comment]
> In both cases, you are uploading the debug files via the `sentry-cli`, yes?

That's right. Did you mean that I still have to upload electron symbol files, although Electron is BUILT-IN REPOSITORIES?
